.azuredevops/security-tools.yml

```diff
@@ -0,0 +1,55 @@
+# Security tools
+
+name: 1.0.$(Year:yy)$(DayOfYear).$(Rev:r) # This is the build number
+
+trigger: none
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily at midnight
+  branches:
+    include:
+    - main
+
+pool:
+  name: dynamics-bc-managed-xl
+
+resources:
+  repositories:
+  - repository: PipelineTemplates
+    type: git
+    name: Infrastructure-PipelineTemplates
+    ref: master
+
+variables:
+- template: security-tools-full-variables.yml@PipelineTemplates
+- template: ./variables-common.yml
+
+jobs:
+- job: default_job
+  displayName: Run security tools
+  timeoutInMinutes: 480
+  steps:
+  - checkout: self
+    fetchTags: 'true'
+  - powershell: |
+      Import-Module $(Build.SourcesDirectory)\Build\Scripts\EnlistmentHelperFunctions.psm1
+      $repoVersion = Get-ConfigValue -Key 'repoVersion'
+      $latestVersion = (Find-Package "microsoft-ALAppExtensions-Modules-preview" -Source "nuget.org" -AllVersions -MinimumVersion $repoVersion | Select Version -First 1).Version
+      Write-Host "##vso[build.updatebuildnumber]$latestVersion"
+  - task: DownloadPackage@1
+    inputs:
+      feed: '/DynamicsSMBPackages'
+      definition: 'microsoft-ALAppExtensions-Modules-preview'
+      version: $(Build.BuildNumber)
+      downloadPath: $(OutputPath)
+  - template: security-tools-full.yml@PipelineTemplates
+    parameters:
+      OutputPath: $(OutputPath)
+      SourcePath: $(SourcePath)
+      BuildCommand: Powershell.exe -NonInteractive "cd $(Build.SourcesDirectory); .\build.ps1 -ALGoProject 'Test Stability Tools'"
```

Comment:
If the pipeline has to build a random project, can it at least be _System Application_?